data/relatedshows/src/main/java/app/tivi/data/relatedshows/TmdbRelatedShowsDataSourceImpl.kt

```diff
@@ -1,52 +0,0 @@
-/*
```

Comment:
Will un-remove this file once https://github.com/MoviebaseApp/tmdb-api/pull/17 is out